src/ProgressAvatar.js

```diff
@@ -74,37 +74,36 @@ class ProgressAvatar extends React.Component {
         let strokeColor = progressColor || success;
 
         return (
-            <Element { ...rest }>
-                <div style = {{
-                        position: "relative",
-                        padding,
+            <Element { ...rest }
+                display="inline-block"
```

Comment:
Wanted to comment that this interface isn't ideal, you're passing css as props, which you can already to by passing style. I can see where it would be merited.

Reply:
Why does it support both? Just curious, know that it's not part of this pr.

Reply:
Good question, I captured this discussion in issue #84